sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -1621,7 +1621,7 @@ class PreemptedError() {
   // errors have the lowest priority.
   def set(error: Exception with SparkThrowable, priority: Option[Int] = None): Unit = {
     val calculatedPriority = priority.getOrElse {
-      error.getCondition match {
+      error.getErrorClass match {
```

Comment:
Further validation will be double-checked on GA, and once `approved`, I will restore it.

Reply:
https://github.com/panbingkun/spark/actions/runs/11303132501/job/31439885127
<img width="1359" alt="image" src="https://github.com/user-attachments/assets/5a68f2f5-7890-4ffd-b08c-8df1ba532c3d">